services/QuillLMS/lib/tasks/auto_annotate_models.rake

```diff
@@ -4,6 +4,7 @@
 # NOTE: are sensitive to local FS writes, and besides -- it's just not proper
 # NOTE: to have a dev-mode tool do its thing in production.
 if Rails.env.development?
+  # rubocop:disable Layout/HashAlignment
```

Comment:
I'm showing a `Lint/RedundantCopDisableDirective: Unnecessary disabling of `Layout/HashAlignment` locally.  Does this disabling necessary?

Reply:
same as above, but I think the extra warning is not worth keeping the table, it looks fine as a regular hash too.